labrad/server.py

```diff
@@ -197,6 +200,13 @@ def __init__(self):
         self.signals = []
         self.contexts = {}
 
+        self.__protocol = None
+        self.__async_client = None
+        self.__sync_client = threading.local()
```

Comment:
Double underscore names get "mangled" by adding the name of the class to the front to make them a little more private. I wanted to make it harder for subclasses to accidentally clobber these and break something. We haven't done this sort of thing consistently, however, so maybe it's more distracting than it's worth.